checker/examples/Makefile

```diff
@@ -4,7 +4,6 @@ all: compile
 
 compile:
 	$(JAVAC) *.java
-	$(JAVAC) units-extension/*.java
```

Comment:
The compilation done here would require a class path to the examples directory for the external units qualifiers to compile. Instead, in the make file within units-extension, it does a fresh compilation and runs the tests. That make file is invoked with ant target units-extension-tests.